airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -167,16 +167,16 @@
     }
 
     /**
-     * Start playing from the current playlist
+     * Start/resume playing from the current playlist
      */
     function onStart() {
         if (CastPlayer.castSession) {
             CastPlayer.playCast();
         } else if ($('#audioPlayer').get(0)) {
-            var audioPlayer = $('#audioPlayer');
-            if(audioPlayer.paused) {
-                skip(0, audioPlayer.currentTime);
-            }
+            // Resume playing if the player was paused
+            if ($('#audioPlayer').get(0).src) $('#audioPlayer').get(0).play();
+            // Start the first track if the player was not yet loaded
+            else skip(0);
```

Comment:
Small nitpick, but can we retain the braces with the if/else branching? I'd like to keep with the existing style.

Reply:
Yes, done!